docs/en/data_types/datetime.md

```diff
@@ -1,15 +1,58 @@
 # DateTime {#data_type-datetime}
 
-Date with time. Stored in four bytes as a Unix timestamp (unsigned). Allows storing values in the same range as for the Date type. The minimal value is output as 0000-00-00 00:00:00.
-The time is stored with accuracy up to one second (without leap seconds).
+Data structure storing Unix timestamp (`UInt32`) and a time zone.
```

Comment:
Unix timestamp is not UInt32.

Reply:
This data structure does not store the time zone for each value. Clarification needed.

Reply:
The DataTime data type is declared as 
```
class DataTypeDateTime final : public DataTypeNumberBase<UInt32>
```
Doesn't it mean that we store Unix timestamp in the `UInt32`-type value?